module-2/backend/week_one.md

```diff
@@ -5,30 +5,58 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  Get, Post, Redirect, Render, Put

```

Comment:
GET, POST, PUT, PATCH, DELETE are the most common HTTP verbs. what's the purpose of each of these?